dashboard/app/admin.go

```diff
@@ -120,6 +121,86 @@ func dropEntities(c context.Context, keys []*db.Key, dryRun bool) error {
 	return nil
 }
 
+func restartFailedBisections(c context.Context, w http.ResponseWriter, r *http.Request) error {
+	if accessLevel(c, r) != AccessAdmin {
+		return fmt.Errorf("admin only")
+	}
+	ns := r.FormValue("ns")
+	if ns == "" {
+		return fmt.Errorf("no ns parameter")
+	}
+	var jobs []*Job
+	var jobKeys []*db.Key
+	jobKeys, err := db.NewQuery("Job").
+		Filter("Finished>", time.Time{}).
+		GetAll(c, &jobs)
+	if err != nil {
+		return fmt.Errorf("failed to query jobs: %w", err)
+	}
+	toReset := []*db.Key{}
+	for i, job := range jobs {
+		if job.Namespace != ns {
+			continue
+		}
+		if job.Type != JobBisectCause && job.Type != JobBisectFix {
+			continue
+		}
+		if job.Error == 0 {
+			continue
+		}
+		errorTextBytes, _, err := getText(c, textError, job.Error)
+		if err != nil {
+			return fmt.Errorf("failed to query error text: %w", err)
+		}
+		fmt.Fprintf(w, "job type %v, ns %s, finished at %s, error:%s\n========\n",
```

Comment:
Do we want to reset some particular errors? Would a regexp parameter to match against the error be useful?

Reply:
At least at the moment there's not much sense to do so, I'll add it once needed.